lib/ebaze_web/controllers/user_controller.ex

```diff
@@ -0,0 +1,62 @@
+defmodule EbazeWeb.UserController do
```

Comment:
How is authentication happening? Should anyone be able to see all the user accounts and delete them?

Reply:
added login authentication commit 56f836ebee36d5a30b67ab6136c527839e26c4e0

Reply:
Okay so this verifies the user on login...what checks if they are actually logged in upon say refresh?

Reply:
there should be a token that verifies that the session is active - is that something that would be implemented with auth?

Reply:
Yes, that would need to be setup!

Reply:
Er, rather, you would need to check for the presence of it to make sure they can only visit pages if logged in

Reply:
adding a check for get_session(conn, :cookies) should work I think!